CONTRIBUTING.md

```diff
@@ -60,4 +60,22 @@ post_processor.fit(X, Y, sensitive_features=sensitive_features)
 post_processor.predict(X, sensitive_features=sensitive_features)
 post_processor._pmf_predict(X, sensitive_features=sensitive_features)
 ```
-</div>
\ No newline at end of file
+</div>
+
+
+# Creating new releases
+
+If you are one of the current maintainers of this project, follow this checklist to create a new release:
+
+1. Create a branch for the release: `git checkout -b release-vxx.xx`
+1. Ensure that all builds run successfully on all operating systems and python versions on that branch
+1. Bump the module version in `fairlearn/__init__.py`
+1. Make a pull request to fairlearn/fairlearn
+1. Merge fairlearn/fairlearn pull request
+1. Tag and push: `git tag vxx.xx; git push --tags`
```

Comment:
Should we have an annotated tag `git tag -a xxxx -m "Text here"` to help a little?

Reply:
what would the annotation be? The tagged version should have CHANGES.md which should then be the explanation